src/genericitems/list.cpp

```diff
@@ -48,6 +48,15 @@ void GenericList::remove(std::string_view name, std::string_view prefix)
     items_.erase(it);
 }
 
+// Remove all items with specified prefix
+void GenericList::removeWithPrefix(std::string_view prefix)
+{
+    auto delimitedPrefix = fmt::format("{}//", prefix);
+    for (auto it = items_.begin(); it != items_.end(); ++it)
+        if (DissolveSys::startsWith(it->first, delimitedPrefix))
+            items_.erase(it);
```

Comment:
```suggestion
    items_.erase(std::remove_if(items_.begin(), items_.end(), [&delimitedPrefix](const auto it){return DissolveSys::startsWith(it.first, delimitedPrefix);}), items.end());
```

Just moving to an algorithm.